tests/v2/validation/extensions/provisioning/config.go

```diff
@@ -0,0 +1,178 @@
+package provisioning
```

Comment:
We should be move this file to the extensions in the framework or atleast the configs that framework uses. See my comments where I noted this